environment.go

```diff
@@ -774,9 +774,17 @@ func (e *envState) ClientConfig() *ClientConfig {
 }
 
 type BuildBaronProject struct {
+	// todo: after migrating the data, switch the logic to use this boolean and not the list
+
+	// since we are now also using this struct for task annotations (including the TicketCreateProject field),
+	// we can no longer assume that the presence of a project in the list means that the user wants to run the build baron.
+	showBuildBaron bool `mapstructure:"show_build_baron" bson:"show_build_baron"`
```

Comment:
I don't understand what we would do if this is set to true. I thought once task annotations goes live we will stop displaying the build baron tab.

Reply:
The build baron will still be displayed in the case when the build baron is configured and a task annotation has not been sent.

Reply:
removed in favor of using TicketSearchProjects